src/MF_CustomDevice/CustomDevice.cpp

```diff
@@ -84,4 +84,20 @@ namespace CustomDevice
         customDevice[device].set(messageID, output);      // send the string to your custom device
     }
 
-} // end of namespace
\ No newline at end of file
+    /* **********************************************************************************
+        This function is called if the status of the PowerSavingMode changes.
+        'state' is true if PowerSaving is enabled
+        'state' is false if PowerSaving is disabled
+        MessageID '-2' for the custom device  for PowerSavingMode
+    ********************************************************************************** */
+    void PowerSave(bool state)
+    {
+        for (uint8_t i = 0; i < customDeviceRegistered; ++i) {
+            if (state)
+                customDevice[i].set(-2, "1");
```

Comment:
It would be great to have const defines or variables with readable labels for the built-in message ids `-1` and `-2`

Reply:
Changed to `MESSAGEID_POWERSAVINGMODE`

Reply:
cool! let's do the same for -1

Reply:
Hmhm, there is no `-1` (Stop of connector)
This message is coming from the connector and directly passed to the custom device.
I see no need to do it here.